charts/oauthproxy/templates/ingress.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if .Values.ingress.enabled -}}
+{{- $svcName := printf "%s-%s" .Values.service.name .Release.Name -}}
+{{- $svcPort := .Values.service.port -}}
+{{- $ingressHost := .Values.ingress.host -}}
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
```

Comment:
Should this chart be accessed without a domain because it will be accessed only internally in the cluster?
suggestion: accessing it through cluster IP

Reply:
It can be accessed through cluster IP or domain. The option is available to use it from outside the cluster

Reply:
We need to define the `oauth_server_url` before building the application binary, or we should add a welcome page to enter the OAuth domain or cluster IP.

Reply:
from what I understand for the possible flow, each application/solution will have to take the `oauth_server_url` as a configurable input(instead of it being hardcoded to oauth.threefold.io ) when installing it on the cluster. And that will be read when starting the application and used accordingly